keyboards/keebio/iris/keymaps/bwprobably/keymap.c

```diff
@@ -0,0 +1,118 @@
+// Copyright 2022 bwprobably
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+
+#include QMK_KEYBOARD_H
+
+extern keymap_config_t keymap_config;
+
+#define _QWERTY 0
+#define _LOWER 1
+#define _RAISE 2
+#define _ADJUST 16
```

Comment:
This is why we use enums, it's much harder to do this. 

Specifically, setting the adjust layer to 16, you're wasting a bunch of space. Layers 3-15 are created, and use up firmware space. 

Also, because the layer mask is 16 bit, the adjust layer wouldn't work properly.
```suggestion
enum custom_layer {
    _QWERTY,
    _LOWER,
    _RAISE,
    _ADJUST,
};
```